language-server-simulator/build.gradle

```diff
@@ -0,0 +1,143 @@
+description = 'Ballerina Language Server Simulator'
+
+apply from: "$rootDir/gradle/javaProject.gradle"
+
+configurations {
+    ballerinaDistribution
+    jBallerinaDistribution
+    dependency {
+        transitive true
+    }
+}
+
+ext {
+    distributionDir = "distribution"
+    nbalSourceDir = "nBallerinaSrc"
+    fhirSourceDir = "fhirSrc"
+    shortVersion = "${version}".split("-")[0]
```

Comment:
@malinthar I think we could get rid of short version and just use the project version. This was used in pre swanlake releases

Reply:
![image](https://github.com/ballerina-platform/ballerina-distribution/assets/35211477/c5adb411-c257-4f92-826c-175c7f031b39)

We have to use the short-version to refer to the highlighted folder above